### PR TITLE
[Advanced-search page] add new behaviour : remove all sort info on scope change

### PR DESCRIPTION
**Pull request by sgoretti**
When the scope changes, the sort info has lots of chance of not being supported by the new scope.
Unless all scopes have the same fields, but it's unlikely.
so we delete the sorting info.

### CODE REVIEW

File: src/page/search/advanced-search/index.js [hunk 1]
```diff
@@ -152,6 +157,13 @@ const AdvancedSearch = {
         this.setState(this._getNewStateFromStore());
     },
     /**
+     * Scope changed, need to remove all existing sort.
+     */
+	_onScopeChange: function _onScopeChange() {
```
Comment (pierr) [inline src/page/search/advanced-search/index.js]: you cannot do that! You need to use the dispatcher...

**Comment by sgoretti** [comment]
ok, ok, mea culpa, I saw these methods, I thought it would be so simple to use them :/

is that ok now?

**Comment by pierr** [comment]
@sgoretti My comment was not a reproof.
It seems good to me, is it tested on your project ?
Thanks.

**Comment by sgoretti** [comment]
Tested and working

**Comment by Tommass** [comment]
@sgoretti can you rebase this branch ?

**Comment by Bernardstanislas** [comment]
@sgoretti still waiting for your rebase :santa:

**Comment by Bernardstanislas** [comment]
Cleaned in #957

**Pull request closed without merging**